packages/engine/src/simulation/package/state/packages/behavior_execution/reset_index_col.rs

```diff
@@ -1,15 +1,9 @@
 use arrow::datatypes::DataType;
 
 use super::*;
-use crate::{
-    datastore::{
-        arrow::batch_conversion::{new_buffer, new_offsets_buffer},
-        batch::iterators,
-    },
-    simulation::package::state::packages::behavior_execution::config::BehaviorId,
-};
+use crate::datastore::arrow::batch_conversion::new_buffer;
 
-pub fn reset_index_col(state: &ExState, behavior_index_col_index: usize) -> Result<StateColumn> {
+pub fn reset_index_col(_state: &ExState, behavior_index_col_index: usize) -> Result<StateColumn> {
```

Comment:
Doesn't seem to be a trait, so again lets delete
```suggestion
pub fn reset_index_col(behavior_index_col_index: usize) -> Result<StateColumn> {
```

@litvand, do we think we're missing something here? Any idea why state was originally included

Reply:
I am unable to suggest a change on this part of the code but a related change will need to be made manually:

```
    fn reset_behavior_index_col(&mut self, state: &mut ExState) -> Result<()> {
        let behavior_index_col = reset_index_col(state, self.behavior_index_col_index)?;
```
|
V
```
    fn reset_behavior_index_col(&mut self) -> Result<()> {
        let behavior_index_col = reset_index_col(self.behavior_index_col_index)?;
```